weave

```diff
@@ -2184,7 +2059,7 @@ EOF
         [ $# -eq 1 ] || usage
         CONTAINER=$(container_id $1)
         ipam_cidrs lookup $CONTAINER $CIDR_ARGS
-        with_container_netns $CONTAINER detach $ALL_CIDRS >/dev/null
+        util_op detach-container $CONTAINER $ALL_CIDRS
```

Comment:
My reasoning was that I didn't code it to send anything to `stdout` so there was no reason to suppress it.

But equally there is no problem suppressing it.